wagtail/admin/templates/wagtailadmin/home/locked_pages.html

```diff
@@ -31,7 +31,11 @@
                             </div>
                             <ul class="actions">
                                 {% if can_remove_locks %}
-                                    <li><button data-action-lock-unlock data-url="{% url 'wagtailadmin_pages:unlock' page.id %}" class="button button-small button-secondary">{% trans "Unlock" %}</button></li>
+                                    <li><button class="button button-small button-secondary"
```

Comment:
If you qr going to move the data attributes to their own line here - it might make sense to move the button to its own line also.

Reply:
Yes